lua/lv-which-key/init.lua

```diff
@@ -198,27 +178,26 @@ if O.extras then
         t = {"<cmd>VimtexTocToggle<cr>", "Toggle Table Of Content"},
         v = {"<cmd>VimtexView<cr>", "View PDF"}
     }
+    normalMappings["p"] = {"<cmd>lua require'telescope'.extensions.project.project{}<cr>", "Projects"}
+    normalMappings["r"] = {
+        name = "Replace",
+        f = {"<cmd>lua require('spectre').open_file_search()<cr>", "Current File"},
+        p = {"<cmd>lua require('spectre').open()<cr>", "Project"}
+    }
+    normalMappings["z"] = {
```

Comment:
I will stop commenting for every plugin that's not part of extra, please review your work again and only configure extra plugins in this condition.